### PR TITLE
Update Pauper Definitions

### PR DESCRIPTION
**Pull request by Spigushe**
Changed a few parsers to reflect changes since July 15th

### CODE REVIEW

**Comment by Spigushe** [comment]
Fixed a bug with Atog/Affinity

File: Formats/Pauper/Archetypes/Burn.json [hunk 1]
```diff
@@ -3,8 +3,16 @@
     "IncludeColorInName": false,
     "Conditions": [
         {
-            "Type": "InMainboard",
-            "Cards": ["Chain Lightning"]
+            "Type": "OneOrMoreInMainboard",
+            "Cards": ["Chain Lightning","Bump in the Night"]
```
Comment (Spigushe) [inline Formats/Pauper/Archetypes/Burn.json:7]: Non intended change:```suggestion
            "Type": "InMainboard",
            "Cards": ["Chain Lightning"]
```

File: Formats/Pauper/Archetypes/Tireless_tribe.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "Name": "Tireless Tribe",
+    "IncludeColorInName": false,
+    "Conditions": [
+        {
+            "Type": "InMainboard",
+            "Cards": ["Tireless Tribe"]
+        }
+    ]
+}
```
Comment (Spigushe) [inline Formats/Pauper/Archetypes/Tireless_tribe.json:10]:
```suggestion
}

```

**Comment by Badaro** [comment]
Let me know when you're done making changes so I can review this PR.

**Comment by Spigushe** [comment]
Yes, I'm currently finishing the last 39 unknown from SNC meta

**Review by Spigushe — COMMENTED** [review]
Update finished for SNC meta except for:
https://magic.wizards.com/en/articles/archive/mtgo-standings/pauper-league-2022-05-18#ponderouscz_-
https://magic.wizards.com/en/articles/archive/mtgo-standings/pauper-league-2022-06-15#outzeroo_-
https://magic.wizards.com/en/articles/archive/mtgo-standings/pauper-league-2022-07-13#ponderouscz_-

I think we're good now

**Comment by Spigushe** [comment]
More than 100 conflicts to fix, I close the PR temporarily

**Pull request closed without merging**